.gitmodules

```diff
@@ -0,0 +1,24 @@
+[submodule "balena-yocto-scripts"]
+	path = balena-yocto-scripts
+	url = https://github.com/balena-os/balena-yocto-scripts
+[submodule "layers/meta-balena"]
+	path = layers/meta-balena
+	url = https://github.com/balena-os/meta-balena
+[submodule "layers/meta-balena-odyssey-x86"]
+	path = layers/meta-balena-odyssey-x86
```

Comment:
Can you please add the git submodule with the same name as the BSP github repo? It would be `meta-odyssey-x86`